src/Tools/dotnet-gcdump/DotNetHeapDump/DotNetHeapDumpGraphReader.cs

```diff
@@ -0,0 +1,992 @@
+using Graphs;
```

Comment:
Missing license header?

Reply:
This is one of the copied files.  The originals did not have license headers.  In this case, I opted for a README that correlates the file with its original and documents any changes.  Let me double check the license on the PerfView repo and make sure to add it to our ThirdPartyNotices file.